src/commands/appSettings/connectionSettings/azureWebJobsStorage/StorageConnectionListStep.ts

```diff
@@ -46,7 +46,7 @@ export class StorageConnectionListStep<T extends IStorageConnectionWizardContext
 
         buttons.push(skipForNow);
 
-        const message: string = localize('connectAzureWebJobsStorage', 'In order to proceed, you must connect a storage account for internal use by the Azure Functions runtime.');
+        const message: string = localize('connectAzureWebJobsStorage', 'Azure Functions needs to be configured to use a storage account.');
```

Comment:
This sentence is really weird to me. It sounds like Azure Functions needs to be configured before you can use a storage account.

Reply:
Maybe, "Azure Functions requires a storage account to be configured."

Reply:
Or maybe "Configure a storage account to be used with your function runtime?"

Reply:
I like that second one